src/Filters/remez_fir.jl

```diff
@@ -347,6 +347,59 @@ plot(f, 20*log10.(abs.(freqz(b,f,1.0))))
 plot(f, 20*log10.(abs.(freqz(b2,f,1.0))))
 grid()
 ```
+
+# Examples from the unittests - standard (even) symmetry.
```

Comment:
Would it be worth it to make these doc-tests? The syntax for doing so is [number six in this list](https://docs.julialang.org/en/v1/manual/documentation/index.html).